Backoffice/Manager/SiteManager.php

```diff
@@ -26,12 +28,16 @@ public function __construct($siteClass)
      */
     public function initializeNewSite()
     {
+        $siteAliasClass = $this->siteAliasClass;
+        $siteAlias = new $siteAliasClass();
```

Comment:
It should be verified here that the object implement the SiteAliasInterface.

Plus I think it will be better to inject a SiteAliasFactory rather than a classname in that context. The factory will be a service easily overridable by an integrator to inject another type of SiteAlias